src/components/HURUmap/Tutorial/Line.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+function Line({ start, end, ...props }) {
```

Comment:
```suggestion
function Line({ x1, y1, x2, y2, ...props }) {
```
Or is there any reason of maintaining `start`/`end` from this point on?